app/main/views/frameworks.py

```diff
@@ -185,7 +185,7 @@ def framework_supplier_declaration(framework_slug, section_id=None):
     ), status_code
 
 
-@main.route('/frameworks/<framework_slug>/<path:filepath>', methods=['GET'])
+@main.route('/frameworks/<framework_slug>/files/<path:filepath>', methods=['GET'])
```

Comment:
This is going to break the old URLs. I think we hard link to them:
- in the declaration (which users can’t see any more)
- in the updates PDFs
- possibly in some of the responses that Nicole has sent out (I'll check)